src/frontend/LayoutViewer.cpp

```diff
@@ -17,25 +17,18 @@
  */
 
 #include <QCloseEvent>
-#include <QPushButton>
 #include <zstd.h>
 #include "LayoutViewer.h"
 #include "Settings.h"
 #include "AboutFile.h"
 #include "base.hpp"
 #include "ui_LayoutViewer.h"
 
-LayoutViewer::LayoutViewer(QString iconTheme, QWidget *parent) :
+LayoutViewer::LayoutViewer(QWidget *parent) :
     QDialog(parent),
     ui(new Ui::LayoutViewer) {
   ui->setupUi(this);
   aboutDialog = new AboutFile(this);
-  auto set_icon = [&](QPushButton* obj, QString icon) {
-    obj->setIcon(QIcon(":/images/" + iconTheme + "/" + icon + ".png"));
-  };
-  set_icon(ui->buttonAboutLayout, "description");
```

Comment:
Why have you removed `ui->buttonAboutLayout` also? :sweat_smile: I just told you about removing `ui->viewAltGr` and `ui->viewNormal` because those two don't need icons anymore, I just forgot to remove the icons when I updated the layout.

Reply:
@ahmubashshir